Add FrauenLoop.

camilasan opened the pull request:
Hi, what do you think? FrauenLoop does a great job on teaching programming and helping women to find jobs in Berlin, specially immigrants.

sujinleeme commented:
Hello. @camilasan. 🌻   Belated IWD! Thanks for your PR. I think it would be good to move FraudenLoop to the Coding Bootcamp section.  Maybe we can elaborate more; it's for women and offers free/paid courses. WDYT :)

camilasan commented:
> Hello. @camilasan. sunflower Belated IWD! Thanks for your PR. I think it would be good to move FraudenLoop to the Coding Bootcamp section. Maybe we can elaborate more; it's for women and offers free/paid courses. WDYT :)

hello :) I moved it and added description.

sujinleeme commented:
@camilasan Thank you so much! I am happy to ship your contribution. Have a nice day. 💐
